### DISCUSSION TITLE
Firefox + websocket + HTTP/2 = The connection to wss ... was interrupted while the page was loading

### DISCUSSION THREAD

**Discussion by marek141017** [tutao/tutanota#6440] [General]
Hello.
I recently noticed that my web interface got disconnected too often.
In firefox log I found this error: The connection to wss ... was interrupted while the page was loading
Search engine suggested that firefox doesn't like websockets over HTTP/2 or may be some messages.
I disabled HTTP/2 for websocket and now my web application is always connected.

Question: is it Firefox issue (I use 117) or is it tuta framework issue?

Regards

**Comment by charlag** [comment]
thanks for letting us know, we will look into this. Generally we don't observe this in FF but it might depend on the network too

**Reply by marek141017** [reply]
Hello
Any update on this?
Is it FF issue or back-end JS?

Regards

**Reply by charlag** [reply]
We don't observe it

**Comment by marek141017** [comment]
But I do:
![Screenshot_2024-02-19_17-47-11](https://github.com/tutao/tutanota/assets/32798889/f09ccf53-9182-4786-8f5b-1b63299cfa9a)
